### DISCUSSION TITLE
Working with swipe

### DISCUSSION THREAD

**Discussion by I-Solaris-I** [AvaloniaInside/Shell#58] [Q&A]
Hello. I have problems with swipe on the touchscreen. The same problem occurs on my android. Can I fix this somehow or make the swipe for tabs unavailable at all.

https://github.com/user-attachments/assets/df14c6ad-1b21-4d6a-a4d1-cac296a4ac7a

**Comment by OmidID** [comment]
Hello @I-Solaris-I 
Thanks for using the Shell.

Main tab view using `Carousel` and what I can see you are able to disable it.

https://github.com/AvaloniaUI/Avalonia/discussions/12655

or more in detail:

https://github.com/AvaloniaUI/Avalonia/issues/13729

**Reply by I-Solaris-I** [reply]
The first option disables interactions with elements inside the carousel, this is not the behavior I expect. I need a swipe on the touch screen or a scroll with the "shift" button pressed to take me to a new tab.

**Reply by OmidID** [reply]
Do you mean after you disable it, the tab cannot be change anymore?

**Reply by I-Solaris-I** [reply]
I mean, after disabling, the contents of the tab are not available. For example, I can't use comboBox or write text in a textBox
